shared-libs/transitions/src/transitions/utils.js

```diff
@@ -65,5 +65,19 @@ module.exports = {
 
   hasRun: (doc, transition) => {
     return !!(doc.transitions && doc.transitions[transition]);
+  },
+
+  getDeprecationMessage(name, deprecatedIn, extraInfo) {
+    if (!name) {
+      return;
+    }
+
+    let message = `"${name}" transition is deprecated`;
+
+    if (deprecatedIn) {
+      message += ` in ${deprecatedIn}`;
+    }
+
+    return extraInfo ? message + `. ${extraInfo}` : message;
```

Comment:
You're mixing string concatenation (+) with template literals here (`). 
```
message + `. ${extraInfo}` ;  message += ` in ${deprecatedIn}`;
```
I think it would look cleaner if we stick to just one string concatenation method.